<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchPage</class>
 <widget class="QWidget" name="SearchPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <property name="horizontalSpacing">
    <number>10</number>
   </property>
   <property name="verticalSpacing">
    <number>8</number>
   </property>
   <item row="3" column="0" colspan="4">
    <widget class="QTableView" name="tableFoundResults">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="0" column="1" alignment="Qt::AlignTop">
    <widget class="QLineEdit" name="txtSearchInput">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Search...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" alignment="Qt::AlignTop">
    <widget class="QPushButton" name="btnSearchAdvance">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="accessibleName">
      <string>primary</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../static.qrc">
       <normaloff>:/static/themes/default/img/sidebar-icons/search.png</normaloff>:/static/themes/default/img/sidebar-icons/search.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QLabel" name="lblSearchDir">
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" alignment="Qt::AlignTop">
    <widget class="QPushButton" name="btnBrowseSearchDir">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QWidget" name="advanceSearchPane" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>10</number>
      </property>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="lblKeyword">
        <property name="text">
         <string>Keyword</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignLeft">
       <widget class="QCheckBox" name="checkRegEx">
        <property name="accessibleName">
         <string>circle</string>
        </property>
        <property name="text">
         <string>RegEx</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" alignment="Qt::AlignLeft">
       <widget class="QCheckBox" name="checkInvert">
        <property name="accessibleName">
         <string>circle</string>
        </property>
        <property name="text">
         <string>Invert</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QLabel" name="lblKeyword_2">
        <property name="text">
         <string>Size</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" alignment="Qt::AlignLeft">
       <widget class="QComboBox" name="cmbSearchTypes_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>All</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Files</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Folders</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Symbolic Links</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="cmbSearchTypes">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>All</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Files</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Folders</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Symbolic Links</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../static.qrc"/>
 </resources>
 <connections/>
</ui>
